MekHQ/src/mekhq/gui/dialog/PersonnelMarketDialog.java

```diff
@@ -443,15 +442,18 @@ public boolean include(Entry<? extends PersonnelTableModel, ? extends Integer> e
                             (nGroup == PersonnelTab.PG_DOC && ((type == Person.T_DOCTOR) || (type == Person.T_MEDIC))) ||
                             (nGroup == PersonnelTab.PG_ADMIN && type > Person.T_MEDIC)
                             ) {
-                        return person.isActive();
-                    } else if(nGroup == PersonnelTab.PG_RETIRE) {
-                        return person.getStatus() == PersonnelStatus.RETIRED;
-                    } else if(nGroup == PersonnelTab.PG_MIA) {
-                        return person.getStatus() == PersonnelStatus.MIA;
-                    } else if(nGroup == PersonnelTab.PG_KIA) {
-                        return person.getStatus() == PersonnelStatus.KIA;
+                        return person.getStatus().isActive();
+                    } else if (nGroup == PersonnelTab.PG_RETIRE) {
+                        return person.getStatus().isRetired();
+                    } else if (nGroup == PersonnelTab.PG_MIA) {
+                        return person.getStatus().isMIA();
```

Comment:
Why would these "isMIA/Retired" checks even be necessary? This is the personnel market, for hiring new personnel? Unless I'm missing something.

Reply:
This is required because of how the tabs on this dialog are currently set up (otherwise we end up with '?' selection options). I'm planning a refactor in the near future to fix this requirement.